_website/download/prelude.md

```diff
@@ -75,6 +75,25 @@ maintain backward compatibility from version to version.
 
 # OS-Specific Packages
 
+## Arch Linux
+
+Elvish PKGBUILDs are available in AUR. Install [`elvish`](https://aur.archlinux.org/packages/elvish/) or [`elvish-git` (HEAD)](https://aur.archlinux.org/packages/elvish-git/) using your favorite AUR helper:
```

Comment:
For consistency, add "(latest release)" after the "elvish" link. Also suggest moving "(HEAD)" out of the link.

Reply:
I object to your choice of AUR helper, but add something like this to the end of the paragraph:

```markdown
(below we assume you are using [yay](link to yay))
```

Reply:
I meant "I don't object" :)

Reply:
Actually, I would like to drop this code block to:
- let the user figure out how to add a 3rd-party repository (because they are Arch users!)
- avoid the problem of AUR helper choices